### PR TITLE
Force Link

### PR DESCRIPTION
**Pull request by AlexanderPadayachee**
Force link fixed. Now works with example files on redmine

### CODE REVIEW

**Review by Yc325 — APPROVED** [review]

File: src/View/OpenPatientWindow.py [hunk 1]
```diff
@@ -643,7 +633,7 @@ def force_link_on_nodes(self):
             logging.debug("Force link unsuccessful")
             QMessageBox.about(self, "Force Link",
                               "Force Link Unsuccessful")
-        self.open_patient_window_link_button.setDisabled(True)
+        #self.open_patient_window_link_button.setDisabled(True)
```
Comment (nwilde) [inline src/View/OpenPatientWindow.py:636]: Remove commented code if no longer needed

File: src/View/OpenPatientWindow.py [hunk 2]
```diff
@@ -586,6 +584,7 @@ def force_link_on_nodes(self):
         for the selected nodes, overwrite the RTPlan and RTDose Series
         ID to match the RT image and RT struct
         """
+        #self.open_patient_window_link_button.setDisabled(True)
```
Comment (nwilde) [inline src/View/OpenPatientWindow.py:587]: remove if not needed

File: src/View/OpenPatientWindow.py [hunk 3]
```diff
@@ -509,7 +507,7 @@ def check_selected_items_referencing(self, items):
 
             if series["SR"] and series["SR"].parent() != series["IMAGE"]:
                 return False
-        self.open_patient_window_link_button .setDisabled(True)
+        #self.open_patient_window_link_button .setDisabled(True)
```
Comment (nwilde) [inline src/View/OpenPatientWindow.py:510]: remove if not needed

**Review by nwilde — APPROVED** [review]
don't know how the functionality works but code looks good, just remove the commented lines of code

**Pull request merged**